style.css

```diff
@@ -147,10 +154,9 @@ video {
 	position: relative;
 }
 
-
 .button i {
-	line-height: 80px;
-	font-size: 50px;
+	line-height: 5vw;
+	font-size: 4vw;
```

Comment:
Should probably use `rem` for these sizes - e.g. if you change the width of your browser window manually they shouldn't really change size